frontend/src/axios/client.ts

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const client = axios.create({
+    baseURL: process.env.API_BASE_URL || "http://localhost:5001"
```

Comment:
Not sure where to add this. Here in dockerfile? I usually use .env files but I don't know if we want to bother setting those up.
![Screenshot 2024-02-16 at 12 52 39 AM](https://github.com/lynx-locks/lynx-web/assets/55325093/3a99e578-bcd0-42db-b898-0f548924ffb8)

Reply:
pretty sure @cameronkinsella said he has the env variables already setup, shouldnt need to add anything

Reply:
No need to add it anywhere, we just need to document it as a runtime option.
By default prod will be using localhost:5001, so users would only need to change this if they have some kind of advanced configuration.

Reply:
I see. But either way i feel like leaving it as an environment variable is better?

Reply:
yeah, I agree. I think they way you have it set up is good.
I was just referring to how that env variable should be set. Usually customizable runtime stuff like that would be set by the user instead of in the dockerfile (since the user never actually sees the dockerfile).

From the user's perspective, they would run `docker pull lynx:frontend` and then `docker run -e API_BASE_URL=https://customdomain.com/api lynx:frontend` or something like that.

TLDR: I think what you did here is good, we should just document user customizable env variables somewhere